### PR TITLE
Updates and additions based on my onboarding experience.

### PR DESCRIPTION
**Pull request by craineum**
This PR encompasses all my changes to the guides based on my onboarding experience. There are probably more, but this is a good start so I don't forget anything.

There is a little bit of everything in this PR. Let me know if you want me to separate out different logical pieces into different PRs.

### CODE REVIEW

**Review by blam77 — APPROVED** [review]

File: tools/README.md [hunk 1]
```diff
@@ -6,9 +6,14 @@ common tools found throughout our engineering organization.
...
 * [asdf](http://asdf-vm.com) - Used for managing versions of programming
   languages (e.g. NodeJS, Ruby, etc.)
+* [homebrew](https://brew.sh/) - Package manager for macOS.
 * [iTerm2](https://iterm2.com) - A more advanced terminal emulator for macOS
   than what you get out of the box.
 * [kitty](https://sw.kovidgoyal.net/kitty/) - A fast GPU based terminal emulator
+* [notion](https://www.notion.so/desktop) - The desktop app version for
+  accessing our documentation,
+* [oh my zsh](https://ohmyz.sh/) - Super powered ZSH configuration.
+* [rectangle](https://rectangleapp.com/) - Window manager for mac.
```
Comment (tute) [inline tools/README.md:16]: Do we recommend this as an org? Do we want to recommend them? These feel more personal to me.

File: rituals/kevin.md [hunk 2]
```diff
@@ -0,0 +1,26 @@
...
+
+### Weekly
+* Think about 1-on-1 talking points.
+* Note any retro items.
+
+### Monthly
+* Re-read / Review guides and documentation
+* Check-in on personal rituals and adjust as needed
+
+## Pattern Based
```
Comment (tute) [inline rituals/kevin.md:25]: Is the idea that some of us post these here? Should we do in our Notion spaces or elsewhere?

**Review by tute — COMMENTED** [review]
Left comments in two places, all else looks great to me!

File: rituals/kevin.md (hunk 2 above)
Comment (tute) [reply rituals/kevin.md:25]: I guess this question goes to Anthony, TIL we have that directory! 😁

Comment (craineum) [reply rituals/kevin.md:25]: I'm good to remove this, I have no skin in this game, just saw it. I like the notion idea. @acostanzo ?

File: tools/README.md (hunk 1 above)
Comment (craineum) [reply tools/README.md:16]: I would agree, I will remove.

File: rituals/kevin.md (hunk 2 above)
Comment (acostanzo) [reply rituals/kevin.md:25]: This is so wildly out dated for me anyway. I'm ok with dropping it

**Pull request merged**